src/GHC/All.hs

```diff
@@ -54,7 +53,7 @@ data ParseFlags = ParseFlags
 
 -- | Default value for 'ParseFlags'.
 defaultParseFlags :: ParseFlags
-defaultParseFlags = ParseFlags NoCpp Nothing defaultExtensions [] defaultFixities
+defaultParseFlags = ParseFlags CppSimple Nothing defaultExtensions [] defaultFixities
```

Comment:
perhaps we might prefer something like
```haskell
Cpphs defaultCpphsOptions
        {boolopts=defaultBoolOptions{hashline=False, stripC89=True, ansi=False}
        ,includes = []
        ,preInclude = []
        ,defines = [("__HLINT__","1")]
        }
```

Reply:
I think we always override it before using it? If so, the simpler the value the better. But I wouldn't be averse to what you suggest either.

Reply:
that was my assumption as well (that we override before use) so, "if it ain't broke don't fix it". let's leave it be for now.